ui/Screen/Project/Source/CheckoutButton.svelte

```diff
@@ -7,8 +7,11 @@
 
   import { dismissRemoteHelperHint, settings } from "../../../src/session.ts";
 
+  export let isContributor = false;
```

Comment:
It;s a bit odd to see mention of contributor here, might be better expressed with `isFork` or something along those lines.